exercises/book-store/src/test/java/BookStoreTest.java

```diff
@@ -0,0 +1,128 @@
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+public class BookStoreTest{
+
+	@Test
+	public void OnlyASingleBook(){
+		Integer[] p = {1};
+		List<Integer> books = new ArrayList<>(Arrays.asList(p));
```

Comment:
You can just pass the integers to `Arrays.asList()`.  No need to declare and assign beforehand.  EG:
```Java
List<Integer> books = new ArrayList<>(Arrays.asList(1));
```
or
```Java
List<Integer> books = new ArrayList<>(Arrays.asList(1,2,3));
```